src/electron/main.tsx

```diff
@@ -140,15 +140,17 @@ if (!handleSquirrelEvent()) {
       });
 
       if (dev_config.redux_tool) {
-        BrowserWindow.addDevToolsExtension(
-          path.join(os.homedir(), dev_config.redux_tool),
-        );
+        const reduxDevToolsPath = path.join(os.homedir(), dev_config.react_tool)
+        app.whenReady().then(async () => {
```

Comment:
you can use 

await app.whenReady();
await session.defaultSession.loadExtension(reduxDevToolsPath);

Reply:
fixed